### ISSUE TITLE
Getting 404 on this link https://aws-amplify.github.io/amplify-android/

### ISSUE THREAD

**Issue by iamdhavalparmar** [aws-amplify/amplify-android#1463]
[The site which will open when you click on the view deployment](https://aws-amplify.github.io/amplify-android/)

**Comment by rjuliano** [comment]
Where's the "View deployment" button you're referring to? AWS Console?

**Comment by iamdhavalparmar** [comment]
@rjuliano No, I'm not referring to the AWS console. You will find 'View Deployment' in the Github pages section.

**Comment by eeatonaws** [comment]
Hi @iamdhavalparmar. Where is the Github pages section you're referring to? Could you provide a link to the page with the 'View Deployment' button you're referencing?
